tests.md

```diff
@@ -0,0 +1,212 @@
+# Tests
+
+The WordPress Hosting Team has some tools for hosters to check their infraestructure and improve compatibility with WordPress and publish them on the Host Test Result information page.
+
+## What is it
+
+### Runner
+
+Hosting companies can have hundreds of websites hosted with WordPress, so it's important to make sure your configuration is as compatible as possible with the software.
+
+To verify this compatibility, the [WordPress Hosting Community team](https://make.wordpress.org/hosting/) provides a series of [PHPUnit](https://phpunit.de/) tests with which to check the operation of WordPress in any environment.
+
+### Reporter
+
+The Runner tests generates some reports related to a bot user (a hosting company), and this captures and displays those test results at the [Host Test Result](https://make.wordpress.org/hosting/test-results/) page. 
```

Comment:
'some reports' is quite generic. Should this be specified here?

Reply:
imrpoved :) i hope